helpers/publicarMovimientoDetalle.go

```diff
@@ -93,7 +94,7 @@ func PublicarMovimientosDetalle(idMovProcExterno int) (movimientosDetalleRespues
 		return []models.MovimientoDetalle{}, formatErr
 	}
 
-	// logs.Debug(fmt.Sprintf("cuentasPublicar: %+v", cuentasPublicar))
+	logs.Debug(fmt.Sprintf("cuentasPublicar: %+v", cuentasPublicar))
```

Comment:
Dejar comentado o eliminar
```suggestion
	// logs.Debug(fmt.Sprintf("cuentasPublicar: %+v", cuentasPublicar))
```